app/main.py

```diff
@@ -1,5 +1,67 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
```

Comment:
coords could be both types, so use annotation like this. The same in all other similar cases

```suggestion
            coords: list | None = None
```

Reply:
when i am using annotation like this, tests are failed :(
![image](https://user-images.githubusercontent.com/123767529/219068748-6104c9be-cf91-42fc-b813-efec274b06ba.png)